src/components/Wizard/Wizard.test.js

```diff
@@ -1,29 +1,16 @@
 import React from 'react';
-import renderer from 'react-test-renderer';
-import { Row, Col } from 'react-bootstrap';
-import { Button } from '../Button';
-import { Wizard } from './index';
-
-import {
-  mockWizardItems,
-  mockLoadingContents
-} from './__mocks__/mockWizardItems';
-
-import {
-  renderWizardSteps,
-  renderSidebarItems,
-  renderWizardContents
-} from './__mocks__/mockWizardRenderers';
+import { mount } from 'enzyme';
+import { Button, Row, Col, Wizard } from '../../index';
 
 test('Wizard loading renders properly', () => {
-  const component = renderer.create(
+  const component = mount(
```

Comment:
unless these are supposed to be integration tests each component should be tested individually.  Maintaining these in the future will be hard otherwise. These also don't provide much value since they are so broad and not focused on asserting anything.